packages/location-field/src/index.tsx

```diff
@@ -185,6 +185,9 @@ const LocationField = ({
   const [isFetching, setFetching] = useState(false);
   const [stateMessage, setMessage] = useState(null);
   const [stateValue, setValue] = useState(getValueFromLocation());
+  const [abortControllers, setAbortController] = useState([
+    new AbortController()
```

Comment:
Is there any need to intitalize this with an `AbortController`? I don't see this controller being used anywhere. You can set the type of the state with `useState<AbortController[]>`.

Reply:
Oh thanks for catching this yes this is residue